config/ldes-client/handleStreamEnd.ts

```diff
@@ -0,0 +1,322 @@
+                    persoon:gebruikteVoornaam
+                }   
+                VALUES ?person {
+                    ${subjectsPublic.slice(i, i+PAGE_SIZE).join(' ')}
+                }
+            }
+        `
+        await updateSudo(personPublicDeleteOldDataQuery);
+        const personPublicQuery = `
+            ${PREFIXES}
+            INSERT {
+                GRAPH <http://mu.semte.ch/graphs/lmb-data-public> {
+                    ?person a person:Person;
+                        ?predicatePerson ?objectPerson.
+                }
+            }WHERE {
+                GRAPH <http://mu.semte.ch/graphs/mandaten-staging> {
+                    ?person a person:Person;
+                        ?predicatePerson ?objectPerson.
+                }
+                VALUES ?predicatePerson{
+                    dct:modified
+                    mu:uuid
+                    foaf:familyName
+                    persoon:gebruikteVoornaam
+                }   
+                VALUES ?person {
+                    ${subjectsPublic.slice(i, i+PAGE_SIZE).join(' ')}
+                }
+            }
+        
+        `
+        await updateSudo(personPublicQuery);
+        
+
+        
+    }
+
+    // We move to the person staging because we want to keep all the data, but not influence future public person migration
+    // Person in mandaten staging graph = waiting to be moved to lmb-public
+    // Person in person staging graph = waiting to be moved to lmb-private
+    const moveToPersonStagingQuery = `
+        ${PREFIXES}
+        DELETE {
+            GRAPH <http://mu.semte.ch/graphs/mandaten-staging> {
+                ?person a person:Person;
+                    ?predicatePerson ?objectPerson.
+                ?person persoon:heeftGeboorte ?birthdate.
+                OPTIONAL {
+                    ?subject adms:identifier ?identifier.
+                    ?identifier ?identifierPredicate ?identifierObject.
+                }
+              } 
+        } INSERT {
+            GRAPH <http://mu.semte.ch/graphs/person-staging> {
+                ?person a person:Person;
+                    ?predicatePerson ?objectPerson.
+                ?person persoon:heeftGeboorte ?birthdate.
+                ?birthdate ?birthdatePredicate ?birthdateObject.
+                OPTIONAL {
+                    ?subject adms:identifier ?identifier.
+                    ?identifier ?identifierPredicate ?identifierObject.
+                }
+            }
+        }WHERE {
+            GRAPH <http://mu.semte.ch/graphs/mandaten-staging> {
+                ?person a person:Person;
+                    ?predicatePerson ?objectPerson.
+                ?person persoon:heeftGeboorte ?birthdate.
+                ?birthdate ?birthdatePredicate ?birthdateObject.
+                OPTIONAL {
+                    ?subject adms:identifier ?identifier.
+                    ?identifier ?identifierPredicate ?identifierObject.
+                }
+            } 
+        }
```

Comment:
This query fails on my end, it does not seem to be allowed to use the `OPTIONAL` keyword inside `INSERT` or `DELETE` bodies.

Reply:
Additionally, I do not think we need to store the `identifier` object/relationship